setup.py

```diff
@@ -64,5 +64,6 @@
               "dev": ["check-manifest", "flake8", "pytest", "pytest-cov",
                       "pytest-dependency"],
               "hard-encoding-detection": ["chardet"],
+              "toml": ["toml"],
```

Comment:
Given https://docs.python.org/3.11/library/tomllib.html why not use https://pypi.org/project/tomli instead of https://pypi.org/project/toml ?

Reply:
Okay. :+1: